app/lib/katello/concerns/base_template_scope_extensions.rb

```diff
@@ -20,7 +20,7 @@ def errata(id)
       end
 
       def host_subscriptions(host)
-        host.subscriptions
+        host.subscriptions.map(&:name).join(', ')
```

Comment:
We shouldn't be changimg existing macros, people may rely on their behavior. How about introducing `host_subscriprions_names` tjat would internally call `host_subscriptions`? The same for `host_products_names` below